### ISSUE TITLE
Uploaded PDF Files not opening in browser

### ISSUE THREAD

**Issue by GrayJumba** [bolt/bolt#7959]
I am uploading files successfully onto a bolt cms powered website. However when I try to access the uploaded pdf document via the upload url for the pdf, chrome give me a "Error Failed to load PDF document." On Safari I am getting encoded extext as attached.

This happens for all uploaded pdfs. This is the pdf url 

https://kiraguandmwangi.co.ke/files/docs/1636623051_so-so-def.pdf

<img width="1432" alt="Screenshot 2021-11-11 at 12 50 35" src="https://user-images.githubusercontent.com/15907404/141276836-2e633c86-1dc3-43e5-841e-779b20dc72d3.png">

**Comment by bobdenotter** [comment]
I don't think this is a Bolt bug. It looks like the PDF itself is correct, but the browser sends it with the incorrect mime type header. 
Perhaps you can set this yourself, or you could ask the webhost.
